test/content-services/categoriesApi.spec.ts

```diff
@@ -195,9 +195,14 @@ describe('Categories', () => {
 
     it('should return 201 while linking category if all is ok', (done) => {
         categoriesMock.get201ResponseCategoryLinked('testNode');
-        categoriesApi.linkNodeToCategory('testNode', [{ categoryId: 'testId1' }]).then((response: CategoryEntry) => {
-            expect(response.entry.id).equal('testId1');
-            expect(response.entry.name).equal('testName1');
+        categoriesApi.linkNodeToCategory('testNode', [{ categoryId: 'testId1' }]).then((response) => {
+            if (response instanceof CategoryEntry) {
```

Comment:
I'm wondering if we should have two separated groups of expects because that may cause that alternative path will never tested. If you want to test alternative then it may be better to have two different tests for that to ensure that both are really tested. @DenysVuika what do you think? Should we leave it as it is or better to change?

Reply:
Sure we can test it, I will create another test case to test linking multiple categories

Reply:
Done